### ISSUE TITLE
(10.11 beta) Crash on "Open Cover Art From File"

### ISSUE THREAD

**Issue by bjschafer** [OpenEmu/OpenEmu#1993]
Looks like every time I try to add some cover art ("Add cover art from file"), OpenEmu crashes.  

[Crash log](https://gist.github.com/bjschafer/c91ea33e53667a178ab9)

Note that I'm running on the El Capitan Public Beta, so I don't expect it to magically work, just thought I'd let you know in case it's something that needs to be fixed before the final release of 10.11.

**Comment by mrvacbob** [comment]
We've got us an infinite loop here:
```
221 com.apple.AppKit              	0x00007fff99e8152c -[NSSplitViewController _setupSplitView] + 903
222 com.apple.AppKit              	0x00007fff99e8114d -[NSSplitViewController viewDidLoad] + 170
223 com.apple.FinderKit           	0x00007fff91209b3f -[FI_TSidebarSplitViewController viewDidLoad] + 52
224 org.openemu.OpenEmu           	0x000000010092c653 OE_NSViewController_view + 140
225 com.apple.AppKit              	0x00007fff99e8152c -[NSSplitViewController _setupSplitView] + 903
226 com.apple.AppKit              	0x00007fff99e8114d -[NSSplitViewController viewDidLoad] + 170
227 com.apple.FinderKit           	0x00007fff91209b3f -[FI_TSidebarSplitViewController viewDidLoad] + 52
```

**Comment by cedwardsmedia** [comment]
Seems to be somewhat related to issue #2000 as well. Same thing seems to occur when trying to create or change libraries in-app.

**Comment by loicloic** [comment]
While I can confirm this does happen under the latest El Capitan DP with OpenEmu 1.0.4 as well, it does not crash with a build of the current repo version (912-g476ae22-Release, no change, built against 10.10 SDK).

As such I'm closing this issue as it is more than likely to be fixed in OpenEmu 1.0.5
